ftw/contentpage/tests/test_newsportlet_listing.py

```diff
@@ -64,3 +63,39 @@ def test_rightcolumn_portlet(self, browser):
         browser.find('More News').click()
         self.assertEquals(['Bookings open'],
                           browser.css('h2.tileHeadline').text)
+
+    @browsing
+    def test_newslisting_of_inherited_news_portlet(self, browser):
+        # Create a news portlet on root which will be inherited by the root's
+        # child objects.
+        manager = getUtility(IPortletManager, name='plone.rightcolumn')
```

Comment:
There is a [news portlet builder](https://github.com/4teamwork/ftw.contentpage/blob/master/ftw/contentpage/tests/builders.py#L92) which could probably make this a bit simpler :wink: